src/components/Organization.vue

```diff
@@ -0,0 +1,469 @@
+<!-- ADD_LICENSE_HEADER -->
+<template>
+	<wrapper :wrapped="false" :style="listItemStyle(depth)">
+		<h3>{{ uiTitle }}
+			<span v-if="schema.required" class="require-badge">
+				<b-badge variant="danger">REQUIRED</b-badge>
+			</span>
+		</h3>
+		<div>
+			<p class="ml-4 card-text text-muted" v-if="uiDescription">
+				<sup><font-awesome-icon icon="quote-left" class="text-muted" /></sup>
+				{{ uiDescription }}
+			</p>
+
+			<div
+				v-for="(org, i) in flattened"
+				:key="getKey(i)"
+				class="mb-2"
+			>
+				<div class="refrow">
+					<div class="org-label">
+						{{ new Array(i*4+1).join("&nbsp;&nbsp;") }}
```

Comment:
i wonder if this will leak

Reply:
I had already replaced this with a better solution (found out about "string".repeat function) but forgot to push it